src/app/panels/terminal.js

```diff
@@ -663,17 +668,27 @@ class Terminal extends Plugin {
     }
     return self.commands[name]
   }
-  _shell (script, scopedCommands, done) { // default shell
+  async _shell (script, scopedCommands, done) { // default shell
```

Comment:
Use JSDoc comment synthax, like that tools like VSCode can show it in their tooltips
```javascript
/** The default Shell */
async _shell (....)
```
You can even describe each param, which makes it easier to understand : 
```javascript
/**
 * The default Shell 
 * @params script { string } Script given by the user through the Terminal
 */
async _shell (....)
```